models/src/main/thrift/content/v1.thrift

```diff
@@ -1518,6 +1518,13 @@ struct Debug {
     4: optional string originatingSystem
 }
 
+struct AliasPath {
+
+    1: required string path
+
+    2: required CapiDateTime ceasedToBeCanonicalAt
```

Comment:
CapiDateTime is interesting; is there nothing more global available?
Looks like it's existing though so happy probably consistant with what other fields do.

Reply:
Yes, I just took a lead from elsewhere. I think it's to provide iso8601 formatting as standard.